### PR TITLE
Editorial note cleanup june 23

### PR DESCRIPTION
**Pull request by rachaelbradley**

### CODE REVIEW

File: guidelines/index.html [hunk 1]
```diff
@@ -236,43 +236,42 @@ <h2>Approaches to Conformance</h2>
...
-			<h3>Outcomes</h3>
+
+			<section data-status="developing">
+			<h3>Outcomes and methods</h3>
 				<details class="summary">
 					<summary>What types of tests and scopes are used?</summary>
 					<p>WCAG 3 includes two (2) types of tests which are evaluated:</p>
 					<ul>
-						<li><strong>Quantifiable tests:</strong> Tests where there is a high degree of consistency between test results from different testers. Examples include testing whether certain properties exist in the content or if they match a value specified by the requirement.</li>
+						<li><strong>Quantifiable tests:</strong> Tests where there is a high degree of consistency between test results from different testers. Examples include testing whether certain properties exist in the content or if thfile:///C:/Users/myacc/Documents/GitHub/silver/guidelines/index.html#outcomesey match a value specified by the requirement.</li>
```
Comment (michael-n-cooper) [inline guidelines/index.html:274]: pasto

File: guidelines/index.html [hunk 2]
```diff
@@ -689,7 +685,7 @@ <h2>Glossary</h2>
 				</dd>
 				<dt><dfn data-lt="equity">Equity</dfn></dt>
 				<dd>
-					<p>Equity is the outcome of processes and actions that ensure the spectrum of human reality obtains what is needed to participate, not solely access. As equity relates to WCAG it is about the impact the standards/guidelines have on people with disabilities, along with actually including PWD in the work. </p> <p class="ednote">This definition needs updating as the Equity group does more work. This definition came from the Equity subgroup of AGWG in Q2 2022</p>
+					<p>Equity is the outcome of processes and actions that ensure the spectrum of human reality obtains what is needed to participate, not solely access. As equity relates to WCAG it is about the impact the standards/guidelines have on people with disabilities, along with actually including PWD in the work. </p>
```
Comment (michael-n-cooper) [inline guidelines/index.html:688]: expand PWD

File: guidelines/index.html [hunk 3]
```diff
@@ -767,7 +761,7 @@ <h2>Glossary</h2>
 				<dd>
 					<p>Technology-specific approach to follow a <a>method</a>.</p>
 				</dd>
-				<dd><p class="ednote">Future work on the glossary will better define terms such as publisher, content author, etc.</p></dd>
+				<dd> </dd>
```
Comment (michael-n-cooper) [inline guidelines/index.html:764]: remove empty element

**Review by michael-n-cooper — CHANGES_REQUESTED** [review]
I left a few comments about minor fixes, good to merge after they're fixed.

**Review by jspellman — APPROVED** [review]

**Review by michael-n-cooper — APPROVED** [review]
pushed a small change, now good to go

**Pull request merged**